imgee/forms.py

```diff
@@ -18,7 +19,7 @@ def valid_file(form, field):
 
 
 class UploadImageForm(Form):
-    file = FileField("File", validators=[Required(), valid_file])
+    file_obj = FileField("File", validators=[Required(), valid_file])
```

Comment:
It's okay to call this `file` with a `NOQA` tag, but if we're renaming, maybe something less lazy than `_obj`. It could be `upload_file` for instance.